starlette/routing.py

```diff
@@ -31,6 +31,11 @@ class NoMatchFound(Exception):
     if no matching route exists.
     """
 
+    def __init__(self, name: str, path_params: typing.Dict[str, typing.Any]) -> None:
+        super().__init__(
+            f'No route exists for name "{name}" and params "{path_params}".'
```

Comment:
Should we be concerned about logging the `path_params` **values** here, and actually just log the **keys**?

Reply:
I see it as a local dev tool so then more context it gives to you then it is easier to debug.

Reply:
My question is more related to the fact that `NoMatchFound` will log in stdout, even with `debug=False`.

Reply:
makes sense. do you suggest to show only keys here?

Reply:
Yes, please.